collect_app/src/main/java/org/odk/collect/android/logic/AuditEvent.java

```diff
@@ -119,14 +138,45 @@ public void setLocationCoordinates(String latitude, String longitude, String acc
         this.accuracy = accuracy;
     }
 
+    public void recordValueChange(String newValue) {
+        this.newValue = newValue != null ? newValue : "";
+
+        // Clear values if they are equal
+        if (this.oldValue.equals(this.newValue)) {
+            this.oldValue = "";
+            this.newValue = "";
+        }
+
+        // some answers might contain commas so wrap all of them in quotes just in case
```

Comment:
👍 I wonder whether it's better to only add quotes when there is a comma or to always add them. Something we can ask users in the beta.

Reply:
Typically, CSV only have quotes when commas. I don't think we should deviate from that.

Reply:
https://csvkit.readthedocs.io/en/0.9.1/contributing.html#principles is my source for that.
> Output favors compatability with the widest range of applications. This means that quoting is done with double-quotes and **only when necessary**, columns are separated with commas, and lines are terminated with unix style line endings (“\n”).

Reply:
Yeah, that would make sense. I'll put in a quick PR and @grzesiek2010 can merge if satisfied.